azext_edge/edge/common.py

```diff
@@ -164,6 +164,16 @@ class K8sSecretType(Enum):
     tls = "kubernetes.io/tls"
 
 
+class FileType(ListableEnum):
```

Comment:
In a future PR we could do some restructuring of common enums and align them closer to their usage areas. Like this (among other examples) is focused on assets.